src/js/puzzles.js

```diff
@@ -1,39 +1,60 @@
 var puzzlesArray = [
-    {
-        id: 1,
-        sentence: 'I have to walk my log.',
-        word: 'log',
-        hint: 'animal',
-        answer: 'dog'
-    },
-    {
-        id: 2,
-        sentence: 'I did not hear the phone because I was a sheep.',
-        word: 'a sheep',
-        hint: 'tired',
-        answer: 'asleep'
-    },
-    {
-        id: 3,
-        sentence: 'I will go grocery stopping tomorrow.',
-        word: 'stopping',
-        hint: 'chore',
-        answer: 'shopping',
-    },
-    {
-        id: 4,
-        sentence: 'I like to play with my set.',
-        word: 'set',
-        hint: 'home animal',
-        answer: 'pet',
-    },
-    {
-        id: 5, 
-        sentence: 'My favorite drink is poke.',
-        word: 'poke',
-        hint: 'red and white label',
-        answer: 'coke',
-    },
-]
+  {
+    id: 1,
+    sentence: 'I have to walk my log.',
+    word: 'log',
+    hint: 'animal',
+    answer: 'dog'
+  },
+  {
+    id: 2,
+    sentence: 'I did not hear the phone because I was a sheep.',
+    word: 'a sheep',
+    hint: 'tired',
+    answer: 'asleep'
+  },
+  {
+    id: 3,
+    sentence: 'I will go grocery stopping tomorrow.',
+    word: 'stopping',
+    hint: 'chore',
+    answer: 'shopping'
+  },
+  {
+    id: 4,
+    sentence: 'I like to play with my set.',
+    word: 'set',
+    hint: 'home animal',
+    answer: 'pet'
+  },
+  {
+    id: 5,
+    sentence: 'My favorite drink is poke.',
+    word: 'poke',
+    hint: 'red and white label',
+    answer: 'coke'
+  },
+  {
+    id: 6,
+    sentence: 'I am on a float',
+    word: 'float',
+    hint: 'water',
+    answer: 'boat'
+  },
+  {
+    id: 7,
+    sentence: 'Here is my new hat',
```

Comment:
This sentence already makes sense, so it may be difficult to guess without a hint. Try something like, 'I have to feed my new hat' or something similar so that it is clear.